pkgs/development/python-modules/openbabel-bindings/default.nix

```
@@ -0,0 +1,24 @@
+{stdenv, openbabel, python, buildPythonPackage}:
+
+  buildPythonPackage rec {
+    pname = "openbabel";
+    version = "3.1.1";
+    nativeBuildInputs = [openbabel];
+    pythonMajorMinor = "${python.sourceVersion.major}.${python.sourceVersion.minor}";
+    src = "${openbabel}/lib/python${pythonMajorMinor}/site-packages";
+    # these env variables are used by the bindings to find libraries
+    # they need to be included explicitly in your nix-shell for
+    # some functionality to work (inparticular, pybel).
+    # see https://openbabel.org/docs/dev/Installation/install.html
+    BABEL_LIBDIR = "${openbabel}/openbabel/3.1.0";
+    LD_LIBRARY_PATH = openbabel;
```

```suggestion
    version = "3.1.1";
    
    src = "${openbabel}/lib/python"${python.sourceVersion.major}.${python.sourceVersion.minor}"/site-packages";
    
    nativeBuildInputs = [ openbabel ];
    
    # these env variables are used by the bindings to find libraries
    # they need to be included explicitly in your nix-shell for
    # some functionality to work (inparticular, pybel).
    # see https://openbabel.org/docs/dev/Installation/install.html
    BABEL_LIBDIR = "${openbabel}/openbabel/3.1.0";
    LD_LIBRARY_PATH = openbabel;
```

This is overwriting the LD_LIBRARY_PATH which is not ideal. Please append it instead.